data/README.txt

```diff
@@ -0,0 +1,9 @@
+`data` directory organization:
```

Comment:
Is this a TXT file on purpose? My assumption is no based on what I perceive to be Markdown formatting.